vividus-allure-adaptor/src/main/java/org/vividus/bdd/report/allure/AllureReportGenerator.java

```diff
@@ -130,12 +143,14 @@ private void patchAllureFiles() throws IOException
         String javascriptString = FileUtils.readFileToString(javascriptFile, StandardCharsets.UTF_8);
         String cssString = FileUtils.readFileToString(cssFile, StandardCharsets.UTF_8);
 
+        String brokenStatusColor = "#d35ebf";
         javascriptString = javascriptString
+                .replace("#aaa", brokenStatusColor)
                 .replace("unknown:\"Unknown\"", "unknown:\"Known\"")
                 .replace("\"failed\",\"broken\",\"passed\",\"skipped\",\"unknown\"",
-                        "\"passed\",\"unknown\",\"failed\",\"broken\",\"skipped\"");
+                        "\"failed\",\"unknown\",\"passed\",\"broken\",\"skipped\"");
         cssString = cssString
-                .replace("#ffd050", "#d35ebf")
+                .replaceAll("#ffd050", brokenStatusColor)
```

Comment:
why is `replace` changed to `replaceAll` ?

Reply:
useless change :)